src/projector.ts

```diff
@@ -29,9 +29,17 @@ export interface ProjectorOptions extends ParentMixinOptions<Child>, EventedOpti
 	 *
 	 * Only applies if `autoAttach` is `true`
 	 */
+	// FIXME Is this used? What about replace?
 	append?: boolean;
 }
 
+export interface AttachOptions {
+	append?: boolean;
```

Comment:
Yes, a default behaviour makes more sense.  I guess logically you could want to supply a `tagName` without being explicit about the attachment type.